static/src/stylesheets/module/_tabs.scss

```diff
@@ -142,19 +138,27 @@ category: Common
  */
 
 .tabs__container--multiple {
-    display: table;
-    table-layout: fixed;
     width: 100%;
     margin: 0;
     margin-bottom: 10px;
+    table-layout: fixed;
+    grid-template-columns: repeat(auto-fit, minmax(120px, 1fr));
+    display: table;
+    display: grid;
 
     .tabs__tab {
-        display: table-cell;
-        width: auto;
         float: none;
+        width: auto;
+        max-width: 100%;
+        @include mq(tablet) {
```

Comment:
Nit: usually we add a space before the `@include`

Reply:
okay, thanks!